### PR TITLE
Rename the Validation categories to consistent with the the public doc

### PR DESCRIPTION
**Pull request by cacheung**
Rename the Validation categories to consistent with the the public doc. (MOB-19617)

## Description
Rename the categories to be consistent with the public doc.
Move "Analytics configuration" check to under "Adobe Analytics category".
Remove "Edge Analytics Streaming Validation" in Edge Analytics category, it doesn't apply to Edge Analytics case.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by amadt — APPROVED** [review]

File: plugins/aep-edge-analytics-streaming-validation/index.ts [hunk 1]
```diff
@@ -1,77 +0,0 @@
-/*
```
Comment (dgrossen) [inline plugins/aep-edge-analytics-streaming-validation/index.ts:1]: is there a reason we're deleting the code for this plugin? Deleting the code won't actually remove the plugin

File: plugins/aep-campaign-configuration/plugin.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
 {
-    "category": "AEP Campaign Configuration",
-    "displayName": "Adobe Campaign AEP SDK Extension Configuration",
-    "description": "Verifies the Adobe Campaign AEP SDK Extension has been configured.",
+    "category": "Adobe Campaign Standard Configuration",
```
Comment (dgrossen) [inline plugins/aep-campaign-configuration/plugin.json:2]: Seems really specific. Probably want something like `Adobe Campaign Standard` or `AEP SDK Configuration`

File: plugins/aep-core-configuration/plugin.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
 {
-    "category": "AEP Core Configuration",
-    "displayName": "Adobe Core AEP SDK Extension Configuration",
-    "description": "Verifies the Adobe Core AEP SDK Extension has been configured.",
+    "category": "Adobe Core Configuration",
```
Comment (dgrossen) [inline plugins/aep-core-configuration/plugin.json:2]: What about `Adobe Core` or `AEP SDK Configuration`? I can't imagine we'll have very many validations under these categories

File: plugins/aep-places-configuration/plugin.json [hunk 4]
```diff
@@ -1,10 +1,10 @@
 {
-    "category": "AEP Places Configuration",
+    "category": "Adobe Places Configuration",
```
Comment (dgrossen) [inline plugins/aep-places-configuration/plugin.json:2]: Adobe Places or AEP SDK Configuration

File: plugins/aep-target-configuration/plugin.json [hunk 5]
```diff
@@ -1,10 +1,10 @@
 {
-    "category": "AEP Target Configuration",
-    "displayName": "Adobe Target AEP SDK Extension Configuration",
-    "description": "Verifies the Adobe Target AEP SDK Extension has been configured.",
+    "category": "Adobe Target Configuration",
```
Comment (dgrossen) [inline plugins/aep-target-configuration/plugin.json:2]: Adobe Target or AEP SDK Configuration

**Review by dgrossen — CHANGES_REQUESTED** [review]
Some of the categories seem really specific and we probably want PM to review

File: plugins/aep-edge-analytics-streaming-validation/index.ts (hunk 1 above)
Comment (cacheung) [reply plugins/aep-edge-analytics-streaming-validation/index.ts:1]: We need to remove this plugin under "Edge Analytics" category.  It doesn't apply to Edge Analytics.

Comment (dgrossen) [reply plugins/aep-edge-analytics-streaming-validation/index.ts:1]: Okay, but deleting the code will not remove the plugin from any category. We either need to find a more appropriate category or delete the plugin from the library if it's not needed.

File: plugins/aep-target-configuration/plugin.json (hunk 5 above)
Comment (cacheung) [reply plugins/aep-target-configuration/plugin.json]: Sent the category names to PM and he went through and ok with the latest updated names.

File: plugins/aep-edge-analytics-streaming-validation/index.ts (hunk 1 above)
Comment (cacheung) [reply plugins/aep-edge-analytics-streaming-validation/index.ts]: Revert this back, we will fix this in another PR.

**Review by dgrossen — APPROVED** [review]

**Pull request merged**